tests/unit/utils/Utils.h

```diff
@@ -3,12 +3,11 @@
  *
  * Authors: Giovanni De Toni, Heiko Strathmann
  */
-#ifndef __UTILS_H__
-#define __UTILS_H__
+#pragma once
 
-#include <shogun/lib/SGMatrix.h>
-#include <shogun/lib/SGStringList.h>
-#include <shogun/lib/SGVector.h>
+#include "shogun/lib/SGMatrix.h"
```

Comment:
why this?

Reply:
I believed that its [standard convention](https://stackoverflow.com/a/21594/3656081) - system files <> vs. project files "". Mainly useful since it helps differentiate project includes vs. system includes.
I can revert if Shogun follows a different convention regarding this.

I had made that change earlier when I had added `NeuralLayerUtil`. Decided to leave that in since I felt its a style improvement.

Reply:
yes but in order to compile the shogun tests we add the include path via `-I` so it by definition is a system include. Check out the other headers ... we are doing this everywhere :)

Reply:
Oh ok, thanks for clarifying. I should also reverse the `""` includes I've done in the other classes then.
Ill just revert my changes on this file then. If needed even the `#pragma once` can be added later.